docs/src/DeveloperDocumentation/new_developers.md

```diff
@@ -77,119 +84,57 @@ We have a large codebase for infrastructure in place, comprising at least
  - polytopes, cones, linear programs
  - polyhedral fans
  - ... and MUCH more
-As well as specialised stuff
+As well as specialized stuff
  - polymake
  - Singular
  - Gap
 
-So: please use it. It is safe to assume all can be improved, however, if we try to perfect every
-single line of code again and again, we we won't get anywhere; there is a balance to be found.
-For preference: interoperability is much more
-important than speed, readability is much more important than speed. Having
-said that: of course, sometimes pure speed matters, but not nearly as often as
-people think.
+So: Please use it. It is safe to assume all can be improved, however, if we try
+to perfect every single line of code again and again, we we won't get anywhere;
+there is a balance to be found.
+For preference: interoperability is much more important than speed, readability
+is much more important than speed. Having said that: of course, sometimes pure
```

Comment:
Perhaps we should also add "correctness over speed" ?

Reply:
I added a table with these things. Feel free to expand. ;)